fenix/app/src/androidTest/java/org/mozilla/fenix/ui/robots/ThreeDotMenuMainRobot.kt

```diff
@@ -110,7 +111,7 @@ class ThreeDotMenuMainRobot {
         assertItemContainingTextExists(
             settingsButton(),
         )
-        if (FeatureFlags.print) {
+        if (FeatureFlags.print && FxNimbus.features.print.value().browserPrintEnabled) {
```

Comment:
Kept `FeatureFlags.print` because this feature is still finishing QA. It also seemed like the Nimbus default should be `true`.

Reply:
👍🏽